libs/api/transaction/data-access/src/lib/api-transaction-data-access.service.ts

```diff
@@ -145,7 +145,8 @@ export class ApiTransactionDataAccessService implements OnModuleInit {
     // Solana Transaction
     appTransaction.solanaStart = new Date()
     try {
-      appTransaction.signature = await solana.sendRawTransaction(transaction)
+      const signature = await solana.sendRawTransaction(transaction)
+      appTransaction.signature = signature
```

Comment:
We can probably inline this again: `appTransaction.signature = await solana.sendRawTransaction(transaction)`